sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/DriverRegistry.scala

```diff
@@ -58,5 +59,15 @@ object DriverRegistry extends Logging {
       }
     }
   }
+
+  def get(className: String): Driver = {
```

Comment:
The change seems okay but why do we need to do? It just moves the codes around.

Reply:
I'm actually in favor of this change - DriverRegistry deals with wrapping on register, and this will also let DriverRegistry deal with unwrapping on get. JdbcUtils no longer needs to know about these details - it just needs to know that it should use `DriverRegistry` instead of `DriverManager`.

Reply:
That said, why don't we look up `wrapperMap` before iterating through `DriverManager.getDrivers`?

Reply:
I am okay either way but I more mean that I wanted to know how it relates to the PR description/title and JIRA

Reply:
This block actually contains one-line change:
`case d: DriverWrapper if d.wrapped.getClass.getCanonicalName == className => d.wrapped`

Reply:
oh! got it